package site.dogether.challengegroup.exception;

public class ChallengeGroupNotFoundException extends ChallengeGroupException {

    public ChallengeGroupNotFoundException(final String message) {
        super(message);
    }
}
